src/Files.App/Dialogs/CreateArchiveDialog.xaml

```diff
@@ -0,0 +1,160 @@
+				<ColumnDefinition Width="Auto" />
+			</Grid.ColumnDefinitions>
+			<TextBlock
+				Grid.Column="0"
+				VerticalAlignment="Center"
+				Text="{helpers:ResourceString Name=Name}" />
+			<TextBox
+				x:Name="FileNameBox"
+				Grid.Column="1"
+				Width="260"
+				PlaceholderText="{helpers:ResourceString Name=EnterName}"
+				Text="{x:Bind ViewModel.FileName, Mode=TwoWay}" />
+		</Grid>
+
+		<!--  Archive Options  -->
+		<Grid
+			Padding="12"
+			Background="{ThemeResource CardBackgroundFillColorDefaultBrush}"
+			BorderBrush="{ThemeResource CardStrokeColorDefaultBrush}"
+			BorderThickness="1"
+			ColumnSpacing="8"
+			CornerRadius="4"
+			RowSpacing="12">
+			<Grid.ColumnDefinitions>
+				<ColumnDefinition Width="*" />
+				<ColumnDefinition Width="Auto" />
+			</Grid.ColumnDefinitions>
+			<Grid.RowDefinitions>
+				<RowDefinition Height="Auto" />
+				<RowDefinition Height="Auto" />
+				<RowDefinition Height="Auto" />
+			</Grid.RowDefinitions>
+
+			<!--  Archive Format  -->
+			<TextBlock
+				Grid.Row="0"
+				Grid.Column="0"
+				VerticalAlignment="Center"
+				Text="{helpers:ResourceString Name=Format}" />
+			<ComboBox
+				x:Name="FileFormatSelector"
+				Grid.Row="0"
+				Grid.Column="1"
+				MinWidth="140"
+				ItemsSource="{x:Bind ViewModel.FileFormats}"
+				SelectedItem="{x:Bind ViewModel.FileFormat, Mode=TwoWay}"
+				SelectedValuePath="Key">
+				<ComboBox.ItemTemplate>
+					<DataTemplate>
+						<TextBlock Text="{Binding Label}" />
+					</DataTemplate>
+				</ComboBox.ItemTemplate>
+			</ComboBox>
+
+			<!--  Compression Level  -->
+			<TextBlock
+				Grid.Row="1"
+				Grid.Column="0"
+				VerticalAlignment="Center"
+				Text="{helpers:ResourceString Name=CompressionLevel}" />
+			<ComboBox
+				x:Name="CompressionLevelSelector"
+				Grid.Row="1"
+				Grid.Column="1"
+				MinWidth="140"
+				HorizontalAlignment="Right"
+				DisplayMemberPath="Label"
+				ItemsSource="{x:Bind ViewModel.CompressionLevels}"
+				SelectedItem="{x:Bind ViewModel.CompressionLevel, Mode=TwoWay}"
+				SelectedValuePath="Key" />
+
+			<!--  Splitting Size  -->
+			<TextBlock
+				Grid.Row="2"
+				Grid.Column="0"
+				VerticalAlignment="Center"
+				Text="{helpers:ResourceString Name=SplittingSize}" />
+			<ComboBox
+				x:Name="SplittingSizeSelector"
+				Grid.Row="2"
+				Grid.Column="1"
+				MinWidth="140"
+				HorizontalAlignment="Right"
+				IsEnabled="{x:Bind ViewModel.CanSplit, Mode=OneWay}"
+				ItemsSource="{x:Bind ViewModel.SplittingSizes}"
+				SelectedItem="{x:Bind ViewModel.SplittingSize, Mode=TwoWay}"
+				SelectedValuePath="Key">
+				<ComboBox.ItemTemplate>
+					<DataTemplate>
+						<StackPanel Orientation="Horizontal" Spacing="4">
+							<TextBlock Text="{Binding Label}" />
+							<TextBlock Text="{Binding Separator}" Visibility="{Binding IsDropDownOpen, ElementName=SplittingSizeSelector}" />
+							<TextBlock Text="{Binding Description}" Visibility="{Binding IsDropDownOpen, ElementName=SplittingSizeSelector}" />
```

Comment:
Is description really needed? Files supports all these formats, so it seems a little redundant.

Reply:
The description concerns the type of media (cd, dvd, ...), not the format. I used a description to avoid the text being too long when closed.

![Archive8](https://user-images.githubusercontent.com/46631671/203837925-7dd20c28-5293-4c68-93a1-e3f3cbb415af.png)
![Archive9](https://user-images.githubusercontent.com/46631671/203837929-665bfbc5-32e3-40da-8ee0-7bce30f39e72.png)